packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -21,11 +23,11 @@ def _ensure_routable():
 def test_if_minuteman_routes_to_vip(cluster):
     """Test if we are able to connect to a task with a vip using minuteman.
     """
-    origin_app, origin_uuid = cluster.get_test_app()
+    origin_app, origin_uuid = get_test_app()
     origin_app['portDefinitions'][0]['labels'] = {'VIP_0': '1.2.3.4:5000'}
-    with cluster.marathon_deploy_and_cleanup(origin_app):
-        proxy_app, proxy_uuid = cluster.get_test_app()
-        with cluster.marathon_deploy_and_cleanup(proxy_app) as service_points:
+    with cluster.marathon.deploy_and_cleanup(origin_app):
+        proxy_app, proxy_uuid = get_test_app()
```

Comment:
nit / not yours but typically in python when there's a "throwaway" / variable not going to be used (`proxy_uuid`), generally just assign it to `_`, so this would be `proxy_app, _ = cluster.get_test_app()`. (This and the couple others around it are also suggesting returning the uuid isn't actually the best option, but that would definitely be an out of scope change).